core/system/src/system.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2013-2015, ARM Limited, All Rights Reserved
```

Comment:
Yup, it should be 2016 (but still including the start date. Most of the file is copy-pasted by a file previously copyrighted 2013-2015).